windows/devices/uwb/UwbConnector.cxx

```diff
@@ -21,9 +21,25 @@ using namespace ::uwb::protocol::fira;
 
 namespace uwb
 {
-class RegisteredCallbackToken
+struct RegisteredCallbackToken
 {
-    uint32_t callbackId;
+    virtual ~RegisteredCallbackToken(){};
+};
+
+struct RegisteredSessionCallbackToken : public RegisteredCallbackToken
+{
+    RegisteredSessionCallbackToken(uint32_t sessionId, std::weak_ptr<::uwb::UwbRegisteredSessionEventCallbacks> Callbacks) :
+        sessionId(sessionId),
+        Callbacks(Callbacks){};

```

Comment:
nit: might as well move the callbacks since the expense of a copy has already been incurred. Also for clarity, the argument name should be lowercase. Same for `RegisteredDeviceCallbackToken` ctor.
```suggestion
    RegisteredSessionCallbackToken(uint32_t sessionId, std::weak_ptr<::uwb::UwbRegisteredSessionEventCallbacks> callbacks) :
        sessionId(sessionId),
        Callbacks(std::move(callbacks)){};
```